apps/docs/app/ui/components/overview/Overview.tsx

```diff
@@ -74,27 +71,8 @@ const Overview = () => {
      component.status === undefined ||
      (component.status === "alpha" && featureFlags.alpha))
                         ).map(component => {
-                            const ComponentIcon = OverviewComponents[component.title] || EmptyComponent;
-
                             return (
-
-                                <Card align="start" ghost className="hd-component-overview-item" key={component._id}>
-                                    <Link
-                                        className="hd-component-overview-item__link"
-                                        href={`/components/${component.slug}`}
-                                    >
-                                        <div className="hd-component-overview-item__img">
-                                            <Icon src={ComponentIcon} />
-                                        </div>
-                                        <div className="hd-component-overview-item__caption">
-                                            <Title as="h3"
-                                                level={3}
-                                                className="hd-component-overview-item__title"
-                                            >{component.title}</Title>
-                                            <p className="hd-component-overview-item__description">{component.description}</p>
-                                        </div>
-                                    </Link>
-                                </Card>
+                                <OverviewTile title={component.title as OverviewTileProps["title"]} key={component._id} />
```

Comment:
not sure we need this as down here. Does it work if you type component as OverviewTileProps instead?